### PR TITLE
Issue #60 - card info sidebar CSS changes

### PR DESCRIPTION
**Pull request by szymonmichalak**
PR introduces CSS changes for card info sidebar, as requested in issue #60.

As agreed with @kbdbrnwsk, all the buttons in the application have now the same style.

### CODE REVIEW

**Comment by theanxy** [comment]
Looks better to me, however maybe we could have something lighter and even more appealing? In particular I’m not sure about the logout button as one could say it feels a bit clunky.

As for the typography, light text on dark background (in buttons) will look much nicer when antialiased—e.g.
```
-webkit-font-smoothing: antialiased;
-moz-osx-font-smoothing: grayscale;
```

Cheers

File: app/styles/_common.scss [hunk 1]
```diff
@@ -98,7 +98,65 @@ main {
   padding: 30px 0 0;
   color: $body-text-alt-color-3;
   text-align: center;
-  z-index: 10;
+  z-index: 3;
+}
+
+/*
+ * Buttons
```
Comment (theanxy) [inline app/styles/_common.scss]: Variables should be moved to the beginning of the file

File: app/styles/_common.scss [hunk 2]
```diff
@@ -98,7 +98,65 @@ main {
...
+  cursor: pointer;
+  display: inline-block;
+  font-weight: bold;
+  line-height: 20px;
+  margin-bottom: 0;
+  margin-top: 10px;
+  padding: 7px 1em;
+  text-transform: none;
+  transition: all 0.3s ease 0s;
+  -moz-transition: all 0.3s ease 0s;
```
Comment (theanxy) [inline app/styles/_common.scss]: No need to use prefixes as we use `grunt-autoprefixer`

**Comment by theanxy** [comment]
@szymonmichalak Any luck with updates to styling? Also, added some notes to the code. Thanks!

**Comment by jkbdbrnwsk** [comment]
@Tade0, could you please take care of this one? :)

**Comment by Tade0** [comment]
Rebased onto `develop` for now. There were conflicts, so I save the branch state from before rebasing in case we would like to resolve them differently.

**Comment by theanxy** [comment]
Closing: styling of buttons in sidebar has been fixed in 8f72eae8bf2fa864d45c1cfb8f2717efecc9e0e5

Buttons in header and in admin panel remain unstyled, but:
1. styling these buttons is less important,
2. it would need to be adjusted to recent CSS updates.

**Pull request closed without merging**